src/webrtc/groupCall.ts

```diff
@@ -628,14 +629,24 @@ export class GroupCall extends TypedEventEmitter<GroupCallEvent, GroupCallEventH
      * Room Member State
      */
 
-    private sendMemberStateEvent(): Promise<ISendEventResponse> {
-        const deviceId = this.client.getDeviceId();
+    private getMemberStateEvents(): MatrixEvent[];
+    private getMemberStateEvents(userId: string): MatrixEvent | null;
+    private getMemberStateEvents(userId?: string): MatrixEvent[] | MatrixEvent | null {
+        if (userId != null) {
```

Comment:
A couple of reasons, the first one being that the new style guide says to avoid implicit boolean casting, and the other one being correctness with the type contract: if userId is a string, this will return a `MatrixEvent | null`. However if this line were `if (userId)`, a userId of `""` would cause it to return a list of events instead, breaking that contract.